docs/concepts/ssh-ca.md

```diff
@@ -0,0 +1,165 @@
+
+In short, CA based authentication is where the central authorities' key is
+trusted by all things within the domain for the purposes of authentication.
+Sometimes the CA will use a different key for authenticating hosts vs users, but
+not always.
+
+## The nominal CA authentication flow
+
+### Background
+
+- **Alice** is someone/something that wants access to a system.
+  - Alice could be a person wanting shell access, an automated process
+    updating the system, etc.
+- **Node** is a remote resource. In our case it is likely an ISOBlue Avena
+  host, but could be anything running SSH.
+
+### Flow
+
+_Note: This flow simplifies the SSH process, but is logically consistent. Also,
+remember this flow happens automatically. From the users perspective they do
+nothing but request a certificate followed by the typical `ssh` commands._
+
+1. Alice requests authorization to certain node(s)
+
+   We use the open source [HashiCorp Vault](https://www.vaultproject.io/)
+   software for OATS fleets.
+
+2. If the central authority approves, it provides Alice a certificate of
+   authorization signed by the CA's user key. The certificate essentially
+   contains:
+
+   - A valid date and time range.
+   - A description of the allowed access (via `principals` and `extensions`)
+   - Reference to Alice's public key.
+
+   Note: A certificate is "public" in the sense that it not useful without
+   Alice's private key (which should be strongly protected by Alice). That said,
+   there may not be a good reason to be intensionally publicly share
+   certificates.
+
+   **In short**: Alice has a "ticket" into the system.
+
+3. Alice connects to the node and collects the node's host public key and
+   certificate. Alice verifies that:
+
+   - the node's public key and certificate match
+   - the certificate originated from the CA's host key.
+
+   **In short**: Alice verifies that the node is trusted by the CA.
+
+4. Alice presents her certificate and her part of the session key, encrypted by
+   her public key.
+
+   **In short**: Alice provides proof that she is authorized and starts
+   authentication process.
+
+5. The node verifies that:
+
+   - Alice's public key and certificate match
+   - Alice's certificate originated from the CA's user key.
+
+   If everything is valid, the node sends Alice its part of the session key,
+   encrypted by the node's host private key.
+
+   **In short**: The node accepts Alice's certificate of authorization and
+   completes the authentication process.
+
+6. Both Alice and node decrypt the parts of the session key that they received,
+   using the peer's public key. Each generates the complete session key. The
+   rest of the session will be encrypted using only this session key. Therefore,
+   both sides will only be able to successfully decrypt the session if each
+   computed the same. When this happens, they can both be sure each side
+   is who they say they are, because the parts of the session key where
```

Comment:
More minor grammatical errors/clarifications
```suggestion
   computed the same key. When this happens, they can both be sure each side
   is who they say they are, because the parts of the session key were
```